chained_filter_controller/src/chained_filter_parameters.yaml

```diff
@@ -0,0 +1,33 @@
+chained_filter:
+  input_interfaces:
+    type: string_array
+    default_value: []
+    read_only: true
+    description: "Name of the input state interface"
+    validation: {
+      not_empty<>: []
+    }
+  output_interfaces:
+    type: string_array
+    default_value: []
+    read_only: true
+    description: "Name of the output state interface"
+    validation: {
+      not_empty<>: []
+    }
+  filter_chain: {
+    type: none,
+    description: "Map of parameters that defines a filter chain, containing filterN as key. Valid for all interfaces, overrides <input_interfaces>.filter_chain if set.
+    The fields for each filter are:
+    type: The filter plugin to be loaded
+    name: Actual name semantically describing the filter, e.g., low_pass_filter
+    params: And underlying map of parameters needed for a specific filter, refer to the specific filter documentation."
+  }
+  __map_input_interfaces:
```

Comment:
One doubt, can the parameter have "/" in ROS 2?

Reply:
Ok. I see it here: https://github.com/ros-controls/ros2_controllers/blob/66a760038ca85e801a370017398a889e83b29d47/chained_filter_controller/test/config/test_multiple_chained_filter.yaml

Reply:
Yess, I doubted it as well but it worked <3